pkg/ecdsa/tss/signer.go

```diff
@@ -13,6 +15,8 @@ type ThresholdSigner struct {
 	// thresholdKey contains a signer's key generated for a threshold signing
 	// scheme. This data should be persisted to a local storage.
 	thresholdKey ThresholdKey
+
+	pubKeyToAddressFn func(cecdsa.PublicKey) []byte
```

Comment:
Where do we set this one? Also, we should make sure it's set after client restart as `ThresholdSigner` is unmarshaled from disk storage.